### PR TITLE
Issue 19 - fixing cli

### PR DESCRIPTION
**Pull request by avrahamgoldman**

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const SparkPost = require('sparkpost');
 const sparkpost = new SparkPost(SPARKPOST_API_KEY);
 const subbable = require('./lib/subbable');
 const defaultFromSparkPost = require('./lib/default-from-sparkpost');
+const CLI_NAME = 'sparkpost';
```
Comment (ewandennis) [inline cli.js]: Would `path.basename(process.argv[1])` be a better fit, in case you or someone else wants to rename the command?

File: cli.js [hunk 2]
```diff
@@ -127,9 +128,14 @@ function setOptionDefaults(module) {
 function setCommandDefaults(module) {
   let self = {};
 
+  if (module.usage) {
```
Comment (ewandennis) [inline cli.js]: If we have `${CLI_NAME}` above, do we also need this clause?

File: cli.js [hunk 3]
```diff
@@ -127,9 +128,14 @@ function setOptionDefaults(module) {
...
+    module.usage = module.usage.replace(/\$0/g, CLI_NAME);
+  }
+
   _.defaults(self, module, {
     options: {},
     describe: `${module.command} command`,
+    usage: `Usage: ${CLI_NAME} ${module.command}${ _.keys(module.options).length > 0 ? ' [options]' : ''}`,
```
Comment (ewandennis) [inline cli.js]: I think I might simplify to just say `[options]` - it's understood that if a command takes no options, then none are needed.

File: cli.js (hunk 2 above)
Comment (avrahamgoldman) [inline cli.js]: For the commands usages that use $0 since they don't have access to that variable in the other files like in `commands/inbound-domains.js`

**Pull request merged**